x-pack/plugins/uptime/public/components/settings/default_email.tsx

```diff
@@ -0,0 +1,157 @@
+
+    const emailError = toEmail.length === 0 ? REQUIRED_EMAIL : undefined;
+
+    setErrors({
+      to: getInvalidEmailError(inValidEmail) || emailError,
+      cc: getInvalidEmailError(inValidCcEmail),
+      bcc: getInvalidEmailError(inValidBccEmail),
+    });
+
+    onChange({
+      defaultEmail: {
+        to: toEmail,
+        cc: ccEmail,
+        bcc: bccEmail,
+      },
+    });
+    // we don't need onChange in dependency
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [ccEmail, toEmail, bccEmail]);
+
+  useEffect(() => {
+    setToEmail(value?.to ?? []);
+    setCcEmail(value?.cc ?? []);
+    setBccEmail(value?.bcc ?? []);
+  }, [value]);
+
+  if (
+    !data?.find(
+      (connector) => connectors?.includes(connector.id) && connector.actionTypeId === '.email'
+    )
+  ) {
+    return null;
+  }
+
+  const emailActionType = actionTypeRegistry.get('.email');
+  const ActionParams = emailActionType.actionParamsFields;
+
+  const onEmailChange = (key: string, val: string[]) => {
+    switch (key) {
+      case 'to':
+        setToEmail(val);
+        break;
+      case 'cc':
+        setCcEmail(val);
+        break;
+      case 'bcc':
+        setBccEmail(val);
+        break;
+      default:
+        break;
+    }
+  };
+
+  return (
+    <EuiDescribedFormGroup
+      title={
+        <h4>
+          <FormattedMessage
+            id="xpack.uptime.sourceConfiguration.alertConnectors.defaultEmail"
+            defaultMessage="Default email"
+          />
+        </h4>
+      }
+      description={
+        <FormattedMessage
+          id="xpack.uptime.sourceConfiguration.defaultConnectors.description.defaultEmail"
+          defaultMessage="Email settings required for selected email alert connectors."
+        />
+      }
+    >
+      <ActionParams
+        actionParams={{
+          to: toEmail,
+          cc: ccEmail,
+          bcc: bccEmail,
+        }}
+        errors={errors}
+        editAction={(key, val, index) => onEmailChange(key, val as string[])}
+        showEmailSubjectAndMessage={false}
+        index={1}
+        isLoading={isLoading}
+        isDisabled={isDisabled}
+      />
+    </EuiDescribedFormGroup>
+  );
+}
+
+const getInvalidEmailError = (val?: string) => {
+  if (!val) {
+    return;
+  }
+
+  return i18n.translate('xpack.uptime.sourceConfiguration.alertDefaultForm.invalidEmail', {
+    defaultMessage: '{val} is not a valid email.',
+    values: { val },
+  });
+};
+
+const REQUIRED_EMAIL = i18n.translate(
+  'xpack.uptime.sourceConfiguration.alertDefaultForm.requiredEmail',
+  {
+    defaultMessage: 'To email is required for email connector',
+  }
+);
+
+export const validateEmail = (email: string) => {
+  return String(email)
+    .toLowerCase()
+    .match(
+      /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
```

Comment:
I know there's a lot of regex for validating email addresses. Do you know if kibana has a standard one some where?